runconfig/opts/parse.go

```diff
@@ -521,6 +529,20 @@ func parseSecurityOpts(securityOpts []string) ([]string, error) {
 	return securityOpts, nil
 }
 
+// parse storage options per container into a map
+func parseStorageOpts(storageOpts []string) (map[string]string, error) {
+	m := make(map[string]string)
+	for _, option := range storageOpts {
+		if strings.Contains(option, "=") {
```

Comment:
@runcom correct. having both `:` and `=` was something we overlooked in the past, so we only fall back for "old" / "existing" options, but new options should only accept `=`